docs/CODEOWNERS

```diff
@@ -72,6 +72,9 @@
 # Faxanadu
 /worlds/faxanadu/ @Daivuk
 
+# Final Fantasy (1)
+# /worlds/ff1/ @Rosalie-A
```

Comment:
This shouldn't still be commented, right?

Reply:
when I said I looked at the file, I neglected to mention I have terrible eyesight lol. Fixed.